### DISCUSSION TITLE
how can I use hooks while using cxjs?

### DISCUSSION THREAD

**Discussion by ketankhairnar** [codaxy/cxjs#666] [General]
I do see state management & data-binding is completely handled by cxjs.  Your answer would help me understand it better
Edit: I completely misunderstood it. It's not react but cx is completely new component framework

**Comment by mstijak** [comment]
Yes, CxJS operates on top of React. 

We have our own hooks implementation, but it's rather experimental and not officially supported/documented at the moment - https://github.com/codaxy/cxjs/tree/master/packages/cx/src/hooks.
